src/Relogify/OpponentList.fs

```diff
@@ -2,75 +2,60 @@ module Relogify.OpponentList
 
 open Fabulous
 open Fabulous.XamarinForms
-open Xamarin.Forms
+open Relogify.ApplicationSettings
 open Relogify.Graphql
 
-type Model =
-     { CurrentPlayerName: string
-       OpponentNames: string array }
+type FetchPlayersState =
+    | NotAsked
+    | Fetching of communityName: string
+    | Fetched of playerNames: string list
 
-let initModel =
-    { CurrentPlayerName = "TestPlayer"
-      OpponentNames = [| "PlaceHolder" |] }
+// TODO: Do we need a record here?
+type Model =
+     { FetchPlayersState: FetchPlayersState }
 
 type Msg =
-    | FetchPlayersClicked
-    | PlayersFetched of string array
+    | PlayersFetched of string list
     | FetchPlayerError of string
 
-type CmdMsg = FetchPlayersCmdMsg
+type CmdMsg = FetchPlayersCmdMsg of communityName: string
+
+let initModel (applicationSettings: ApplicationSettings): Model * CmdMsg list =
+    let cmdMsgs =
+        match applicationSettings.PlayerName, applicationSettings.CommunityName with
```

Comment:
Seems like `PlayerName` is not used in any way here, maybe it is enough to just pass `CommunityName`?

Reply:
I will be used in order to filter out the player from the list, so we can keep it now in order to refactor less later.